optaplanner-examples/src/test/java/org/optaplanner/examples/nqueens/score/NQueensConstraintProviderTest.java

```diff
@@ -115,8 +116,8 @@ private static NQueens readSolution(String resource) throws IOException {
     void givenSolutionMultipleConstraints(ConstraintVerifier<NQueensConstraintProvider, NQueens> constraintVerifier)
             throws IOException {
         constraintVerifier.verifyThat()
-                .givenSolution(readSolution("256queensScore-30.xml"))
-                .scores(SimpleScore.of(-30));
+                .givenSolution(readSolution("256queensScore-29.json"))
```

Comment:
Could you please explain how has the score changed?

Reply:
It hasn't. I just didn't catch the solver in time for 30, and I stopped it at 29. This doesn't change anything - it's just a different point in the solver run.

Reply:
There is no reason why the score should be 30. It is an arbitrary point in time. So I just picked a different one.